packages/core/src/index.ts

```diff
@@ -32,5 +35,14 @@ export function analyzeProject(projectDirectory: string = process.cwd()): Projec
   let languageServer = new GlintLanguageServer(glintConfig, documents, transformManager);
   let shutdown = (): void => languageServer.dispose();
 
-  return { glintConfig, transformManager, languageServer, shutdown };
+  return {
+    glintConfig,
+    transformManager,
+    languageServer,
+    documents,
+    service: languageServer.service,
+    shutdown,
+  };
 }
+
+export { DocumentCache, TransformManager, GlintConfig, loadConfig, GlintLanguageServer };
```

Comment:
I will definitely defer to @dfreeman here on whether we want the `loadConfig` exported (I could see it being fine, but I also have an intuitive HMMMMM response); but we *definitely* want to go out of our way here to make sure the others are `export type` rather than direct `export`; they should absolutely never be constructed directly by end users.